app/components/menu-sidebar.hbs

```diff
@@ -88,7 +88,7 @@
       />
 
       {{! Link to alpha-tomato }}
-      <a class='link' href='https://streep.csvalpha.nl' target='_blank'>
+      <a class='link' href={{sofiaUrl}} target='_blank'>
```

Comment:
```suggestion
      <a class='link' href={{this.sofiaUrl}} target='_blank'>
```
https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/#component-templates__template-this